README.md

```diff
@@ -1 +1,64 @@
-# webpack-setup
\ No newline at end of file
+# todo list
+Restructuring the Awesome books app with ES6
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Awesome Books.
+> This is a todo list project
+
```

Comment:
kindly change your project's name to todo-list up not awesome books

Reply:
Alright, thanks! for helping me become a better coder.